Documentation for Medallia destinations connector

muqing-zhou opened the pull request:
Added for review by Adobe documentation team:

- New file "medallia-connector.md" for new destination connector with Adobe Experience Platform (FYI we also have a separate Launch extension in the same VOC folder with file "medallia.md")
- Supporting screenshots

muqing-zhou committed:
Medallia destination connector images

muqing-zhou committed:
Create medallia-connector.md

Adds destination connector page for Medallia to the Adobe documentation repository

vgiurgiu commented on help/destinations/catalog/voice/medallia-connector.md:
```diff
@@ -0,0 +1,108 @@
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Import API Name]**: Name of the Medallia Import API (also known as Web Feed) to be used in this connection. Youcan activate different segments to different Import APIs to trigger different survey programs
```
Minor comment: You can (missing space). And please add a period at the end of the sentence.

muqing-zhou replied:
Thanks for catching! Updated

vgiurgiu commented on help/destinations/catalog/voice/medallia-connector.md:
```diff
@@ -0,0 +1,108 @@
+> Share with your Medallia delivery team the exact **Attribute names** for every target custom attribute you map using **Target field** > **Select custom attributes** > **Attribute name**. You may wish to take a screenshot of the mapping page to share directly.
+
+
+## Exported data {#exported-data}
```
One comment here. The purpose of this section is for readers to verify that the data is indeed being correctly exported to your destination. 

So, for example, you could tell them to go into the Medallia interface and check that the profiles or segments out of Experience Platform are being populated.

muqing-zhou replied:
Gotcha - updated the section - let me know what you think.

vgiurgiu commented on help/destinations/catalog/voice/medallia-connector.md:
```diff
@@ -0,0 +1,108 @@
+
+
+## Exported data {#exported-data}
+Data is exported in batches of 1000 profiles or 30 minute increments (whichever occurs first) as profiles qualify for your activated segments.
```
Three should also be a Data Governance section in the self-service template. That is boilerplate information for any destination, so please keep it on the page.

muqing-zhou replied:
Added to the end of the template

vgiurgiu reviewed (commented):
@muqing-zhou , great work! Thanks for submitting the documentation. I just left a few small comments. Please let me know if you have any questions.

vgiurgiu commented on help/destinations/catalog/voice/medallia-connector.md:
```diff
@@ -0,0 +1,108 @@
+
+# Medallia connection
+
+## Overview {#overview}
```
Let's please leave a blank line between headings and the paragraph that follows them. This comment applies to all headings in the documentation page. Thanks!

muqing-zhou replied:
Done!

muqing-zhou committed:
Update medallia-connector.md

vgiurgiu commented:
@muqing-zhou  - thanks for making the updates. 

I don't have any other questions at this time. After our engineering team reviews the destination submission and confirms that everything works as expected, we'll be able to publish the documentation as well.

vgiurgiu commented:
@muqing-zhou - we reviewed the destination submission one more time in our team and came across one small thing that I overlooked in my earlier review. 

As per the self-service template (https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#overview ), can you please add a note similar to the one in the Overview section, that mentions that the documentation was authored by Medallia and indicates a support channel where your team can be reached? 

```
>[!IMPORTANT]
>
>This documentation page was created by the *YOURDESTINATION* team. For any inquiries or update requests, please contact them directly at *Insert link or email address where you can be reached for updates*
```

muqing-zhou committed:
Update medallia-connector.md

Added to overview section note on authorship and contact information

muqing-zhou commented:
@vgiurgiu thanks for that callout! added the note to the overview section and our contact info

muqing-zhou committed:
Update screenshots per configuration change

Update to medallia-destination-details.png and medallia-destination-oauth.png

muqing-zhou committed:
Update Medallia screenshot

muqing-zhou committed:
Update medallia-connector.md

Moved API Gateway URL from Authentication section to Destination details section

vgiurgiu commented:
@muqing-zhou - We moved this pull request over into our enterprise Git instance and published the documentation page here: 
https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/voice/medallia-connector.html

Thank you for your work! Closing this PR.
